### ISSUE TITLE
 Fix tab order for citation-related tabs

### ISSUE THREAD

**Issue by Siedlerchr** [JabRef/jabref#10821]
**Is your suggestion for improvement related to a problem? Please describe.**
The tab order in the entry editor should be changed and the citation related items should be grouped together

<img width="1360" alt="grafik" src="https://github.com/JabRef/jabref/assets/320228/59ffe50f-3202-4df8-94da-868695ee06bb">

**Describe the solution you'd like**

The following order should be used (and they should be placed after File Annotation:
1. Scite Tab (Rename tab to "Citation information" and also the setting in the preferences)
2. "Citation relations"
3. "Related articles"

**Additional context**
https://github.com/JabRef/jabref-issue-melting-pot/issues/337

**Comment by brielle5810** [comment]
Hi! I've never contributed to an open-source project before, but would I be able to take up this issue?

**Comment by ThiloteE** [comment]
Sure, go ahead 👍
